### PR TITLE
Remove "bash" highlighting for help output docs

### PR DESCRIPTION
**Pull request by eikowagenknecht**
## Proposed change
The bash type highlighting highlighted every "for", "reboot" etc. which doesn't make sense for this kind of output.
Also rearranged the "backups" config entry to match the colsole output.

## Type of change
The bash type highlighting highlighted every "for", "reboot" etc.
- [X] Spelling, grammar or other readability improvements (`current` branch).
- [ ] Adjusted missing or incorrect information in the current documentation (`current` branch).
- [ ] Added documentation for a new integration I'm adding to Home Assistant (`next` branch).
  - [ ] I've opened up a PR to add logo's and icons in [Brands repository](https://github.com/home-assistant/brands).
- [ ] Added documentation for a new feature I'm adding to Home Assistant (`next` branch).
- [ ] Removed stale or deprecated documentation.

## Additional information

- Link to parent pull request in the codebase: 
- Link to parent pull request in the Brands repository: 
- This PR fixes or closes issue: 

## Checklist

- [X] This PR uses the correct branch, based on one of the following:
  - I made a change to the existing documentation and used the `current` branch.
  - I made a change that is related to an upcoming version of Home Assistant and used the `next` branch.
- [X] The documentation follows the Home Assistant documentation [standards][].

[standards]: https://developers.home-assistant.io/docs/documenting/standards

### CODE REVIEW

File: source/_includes/common-tasks/commandline.md [hunk 1]
```diff
@@ -58,7 +58,7 @@ Replace x.y.z with the desired version like `--version {{current_version}}`
 
 You can get a better description of the CLI capabilities by typing `ha help`:
 
-```bash
+```
```
Comment (frenck) [inline source/_includes/common-tasks/commandline.md]: There must be a language flag set.

**Review by frenck — CHANGES_REQUESTED** [review]

File: source/_includes/common-tasks/commandline.md (hunk 1 above)
Comment (eikowagenknecht) [reply source/_includes/common-tasks/commandline.md]: Why is that? For example in the github docs (https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks) I read "You can add an *optional* language identifier. I coudn't find a markdown guide related to code formatting for home assistant. If there is one, might you point me to it? https://developers.home-assistant.io/docs/documenting/standards doesn't seem to cover this.

Comment (frenck) [reply source/_includes/common-tasks/commandline.md]: This is not GitHub pages or docs. So their documentation does not apply here.
Or CI guards it, to prevent it is begin forgotten (which happened a lot).

In this case, just setting it to `txt` might be fitting.

**Comment by eikowagenknecht** [comment]
I see the language fences are a requirement of the automatically run checks here. Since there are use cases where no code style is clearly preferrable (as with documentation output here) to avoid miscoloring, I suggest adding "{ allowEmpty: true }" as documented here: https://github.com/remarkjs/remark-lint/blob/main/packages/remark-lint-fenced-code-flag/readme.md

**Comment by frenck** [comment]
> to avoid miscoloring, I suggest adding

No, we should not do that, see my comment above.

File: source/_includes/common-tasks/commandline.md (hunk 1 above)
Comment (eikowagenknecht) [reply source/_includes/common-tasks/commandline.md]: Of course I understand that this is not GitHub pages, I referenced it only as a (widely used) example. But I understand why you want to leave this check in the CI. Changing it to "txt" seems to work fine.

File: source/_includes/common-tasks/commandline.md [hunk 2]
```diff
@@ -33,7 +33,7 @@ ha supervisor update
 
 ### Host
 
-```bash
+```txt
```
Comment (frenck) [inline source/_includes/common-tasks/commandline.md]: These are valid shell commands and should therefore not be `txt` but `bash`. Please revert this change.

Comment (eikowagenknecht) [reply source/_includes/common-tasks/commandline.md]: Done. Reason was that the highlighting is kind of weird with the bash markdown style, but I see that this is probably better addressed somewhere else (in the markdown plugin).

**Review by frenck — APPROVED** [review]
Thanks, @eikowagenknecht 👍
